docs/parameters.md

```diff
@@ -0,0 +1,181 @@
+# Working with parameters
+
+It's possible to supply data to actions and these can be provided in a few different ways.  This page outlines how to configure parameters when deploying packages and actions, and how to supply parameters to actions when invoking the action.
```

Comment:
Perhaps an additional note here: actions only have access to parameters and so passing in “environment variables”, or default values or runtime values is done through a single mechanism... something to this affect. Do you know what I mean?